osmosis/assetlist.json

```diff
@@ -14445,6 +14445,50 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/tinkernet/images/tnkr.svg"
         }
       ]
+    },
+     {
+      "description": "YAWP",
+      "denom_units": [
+        {
+          "denom": "ibc/D1DD525F3226643940897A170DFDFC3727F5C4C43131C4D91EEF7D0E93F102B7",
+          "exponent": 0,
+          "aliases": [
+	          "uyawp",
+            "factory/neutron1shu8ygztjpwppfa454c4a74e49erfyus0y432q/YAWP"
+          ]
+        },
+        {
+          "denom": "YAWP",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/D1DD525F3226643940897A170DFDFC3727F5C4C43131C4D91EEF7D0E93F102B7",
+      "name": "YAWP",
+      "display": "YAWP",
+      "symbol": "YAWP",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "neutron",
+            "base_denom": "factory/neutron1shu8ygztjpwppfa454c4a74e49erfyus0y432q/YAWP",
```

Comment:
This asset is not registered in neutron's assetlist. Please register it there as well.

Reply:
Pending PR https://github.com/cosmos/chain-registry/pull/4310